### PR TITLE
Add binary to benchmark model load speed

### PR DESCRIPTION
**Pull request by SS-JIA**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#74700 Add binary to benchmark model load speed**
* #74769 [vulkan] Add option to trigger Vulkan context to load on application start
* #74699 [vulkan] Remove unnecessary include in vulkan_api_test

Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

### CODE REVIEW

**Comment by SS-JIA** [comment]
@SS-JIA has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35124881).

File: binaries/load_benchmark_torch.cc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <string>
+#include <vector>
+
+#include <ATen/ATen.h>
+#include "caffe2/core/timer.h"
```
Comment (beback4u) [inline binaries/load_benchmark_torch.cc:21]: Can we re-organize the order of includes?

**Review by beback4u — APPROVED** [review]

**Comment by SS-JIA** [comment]
@SS-JIA has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35124881).

**Comment by SS-JIA** [comment]
@SS-JIA has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35124881).

**Comment by SS-JIA** [comment]
@SS-JIA has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35124881).

**Comment by SS-JIA** [comment]
@SS-JIA has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35124881).

**Comment by SS-JIA** [comment]
@SS-JIA has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35124881).

**Pull request closed without merging**